packages/protocol/src/data/inscription.ts

```diff
@@ -209,6 +206,11 @@ export interface ContentObject {
 export interface MapStringString {
   [k: string]: string;
 }
+export interface CustomFieldMeta {
+  description: string;
+  name: string;
+  type: string;
```

Comment:
this should maybe more typed?

Reply:
I think we know what different types here exist right?

Reply:
Does not exist anymore 👍. Instead of CustomFieldMeta, there is now customFieldSchema, which is of type "WfCustomField". If you're interested, this is the PR for the core changes.
https://github.com/axonivy/core/pull/5006